Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'turbolinks', '~> 5'
 gem 'bootsnap', '>= 1.1.0', require: false
+gem "haml"
```

Comment:
There’s a Rails version of this gem. https://github.com/indirect/haml-rails Perhaps it’s better suited for you?

Reply:
I saw this but was not sure about it i will take a look now.